### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#493] [Esami & Esoneri M-Z]
### Traccia

<img width="950" alt="image" src="https://github.com/user-attachments/assets/d7b70b5d-6c28-4820-ad8d-706a90d62915">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/e6d9cbd6-a898-45a7-a795-f35dc0d00caf)
